### ISSUE TITLE
Session Directory size too large

### ISSUE THREAD

**Issue by leedavi** [cantarus/PolyDeploy#113]
PolyDeplys keeps a set of installed zip file in the "\DesktopModules\Cantarus\PolyDeploy\Sessions" folder.  

This never seems to get cleaned up and can cause issues with the size of the DNN system, which in turn can cause issues with the backup.

Can this folder be cleaned, so it's only saving required sessions folder?   

It can simply check the session folders when it starts, and delete any older that 2 days?

**Comment by bdukes** [comment]
I agree, and I would like for a step to be added after a successful install that deletes the session.

**Comment by david-poindexter** [comment]
Any work on this should probably be done in the DNN Platform repo since we are working to include this in DNN 10.  There is a branch set up for this effort here:
https://github.com/dnnsoftware/Dnn.Platform/tree/feature/bulk-install
